Examples/SearchExamples/Examples.swift

```diff
@@ -10,6 +10,6 @@ struct Example {
     let screenType: ExampleController.Type
 }
 
-protocol ExampleController: UIViewController {
+public protocol ExampleController: UIViewController {
```

Comment:
```suggestion
protocol ExampleController: UIViewController {
```

Reply:
This was necessary to build the Examples/SearchExample project with Xcode 14.1 on the iOS 13.6 deployment target.